examples/react-router-4/components/App.js

```diff
@@ -28,9 +35,10 @@ function App({ logout }) {
           {' '}
           <Link to="/admin">{'Admin'}</Link>
           {' '}
-          <Link to="/login">Login</Link>
-          {' '}
-          <button onClick={() => logout()}>Logout</button>
+          {user.data ?
+            <button onClick={() => logout()}>Logout</button> :
+            <Link to="/login">Login</Link>
+          }
```

Comment:
I like the idea of hiding the login/logout buttons. Maybe the examples should use the `userAuthWrapper` hoc? The ternary is probably cleaner but then the example can show more of the API